### PR TITLE
zh-TW: cr2 Day 4 translations

### PR DESCRIPTION
**Pull request by henrif75**
Professional translations for CR2 content.
Changes are being staged in a separate branch until all zh-TW content is reviewed.

#1463 #684

### CODE REVIEW

**Comment by johnathan79717** [comment]
Why are there changes to `ja.po` in this PR?

**Comment by henrif75** [comment]
> Why are there changes to `ja.po` in this PR?

Good catch, that was a mistake. I'm fixing it right now, thanks for the heads up!

File: po/zh-TW.po [hunk 1]
```diff
@@ -9207,46 +9205,46 @@ msgstr "錯誤處理：恐慌、`Result`，以及 try 運算子 `?`。"
 #: src/welcome-day-4.md
 msgid ""
 "Unsafe Rust: the escape hatch when you can't express yourself in safe Rust."
-msgstr ""
+msgstr "不安全的 Rust：不能用安全的 Rust 表示時的應急方法。"
```
Comment (mingyc) [inline po/zh-TW.po:9208]:
```suggestion
msgstr "不安全的 Rust：不能寫出安全的 Rust 時的應急方法。"
```

File: po/zh-TW.po [hunk 2]
```diff
@@ -9737,180 +9769,192 @@ msgid ""
...
 
 #: src/testing.md
 msgid "[Compiler Lints and Clippy](./testing/lints.md) (3 minutes)"
-msgstr ""
+msgstr "[編譯器 Lint 和 Clippy](./testing/lints.md) (3 分鐘)"
 
 #: src/testing.md
 msgid "[Exercise: Luhn Algorithm](./testing/exercise.md) (30 minutes)"
-msgstr ""
+msgstr "[練習：盧恩演算法](./testing/exercise.md) (30 分鐘)"
```
Comment (mingyc) [inline po/zh-TW.po:9957]:
```suggestion
msgstr "[練習：盧恩 (Luhn) 演算法](./testing/exercise.md) (30 分鐘)"
```

**Review by mingyc — CHANGES_REQUESTED** [review]

**Review by mingyc — APPROVED** [review]

**Comment by henrif75** [comment]
Thanks @mingyc!

**Pull request merged**